runners/pom.xml

```diff
@@ -39,6 +39,7 @@
     <module>direct-java</module>
     <module>flink</module>
     <module>spark</module>
+    <module>gearpump</module>
```

Comment:
Although Gearpump requires Java 8, I don't think either Beam application or the tests need be compiled in Java 8 to be run on Gearpump. That said, let me try this to make test matrix happy.